### DISCUSSION TITLE
CLI - Export to S3-API Compliant destination

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#718] [Enhancements]
This will allow users to maintain additional copy on a cloud of their choice. 
Consider adding support for client-side encryption support.

**Comment by philhug** [comment]
Not sure if this should be a separate issue, but it would be nice to be able to export everything encrypted (including metadata).
The usecase would be to have automatic backup of ente.io on an untrusted machine locally or to S3.

Then add a separate cli command to decrypt the data.

I have not looked into the code yet, but I assume that the decryption key is derived from the login password, so there might be a need to be able to authenticate without having access to the decryption key, to prevent the backup machine from being able to decrypt the date.
